openedx/adg/lms/applications/serializers.py

```diff
@@ -0,0 +1,147 @@
+"""
+Serializers for applications app
+"""
+
+from django.utils.translation import ugettext_lazy as _
+from rest_framework import serializers
+
+from .models import Education, WorkExperience
+
+# pylint: disable=translation-of-non-string
+
+COMPLETED_DATE_YEAR = 'date_completed_year'
+COMPLETED_DATE_MONTH = 'date_completed_month'
```

Comment:
Please avoid constants for meaningful and simple strings as we read in clean code book.